src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11703,13 +11707,23 @@ namespace UnitarySystems {
                                                             _,
                                                             true,
                                                             this->m_FanOpMode,
-                                                            PartLoadRatio,
+                                                            this->m_SuppHeatPartLoadFrac,
                                                             this->m_SuppHeatingSpeedNum,
                                                             this->m_SuppHeatingSpeedRatio);
             } break;
             default: {
-                if (this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingElectric_MultiStage) {
-                    this->calcMultiStageSuppCoilStageByLoad(state, SuppHeatCoilLoad, FirstHVACIteration);
+                if (this->m_EMSOverrideSuppCoilSpeedNumOn) {
+                    if (SuppHeatCoilLoad > 0.0) {
+                        this->setEMSSuppCoilStagePLR(state);
+                    } else {
+                        this->m_SuppHeatingSpeedRatio = 0.0;
+                        this->m_SuppHeatingCycRatio = 0.0;
+                        this->m_SuppHeatPartLoadFrac = 0.0;
+                    }
```

Comment:
If EMS overrides the SpeedNum the model will only use that SpeedNum of there is a supp heating coil load?

Reply:
Yes I think so. the line 11716 is the condition that only set EMS values when there's a supp heat coil load.